viadot/sources/sap_rfc.py

```diff
@@ -153,6 +153,11 @@ def check_connection(self) -> None:
         self.con.ping()
         self.logger.info("Connection has been validated successfully.")
 
+    def close_connection(self) -> None:
```

Comment:
Should we also/instead add a context manager? See https://github.com/dyvenia/viadot/blob/b565ee4438a901f521affe39b3408044694bf940/viadot/sources/databricks.py#L78-L84 So basically we can then do 
```python
with Databricks() as d:
    d.to_df()
```
Basically, it would be nice to have a general approach to handling sessions/connections in sources, that we could document inside `CONTRIBUTING.md`.